controllers/search/search.go

```diff
@@ -64,6 +64,10 @@ func SearchHandler(c *gin.Context) {
 	if c.Query("order") == "true" {
 		searchForm.SortOrder = true
 	}
+	searchForm.SortType, err = strconv.Atoi(c.Query("sort"))
```

Comment:
It's not that, NewSearchForm have an argument with c *gin.Context. This function should provide a default SearchForm based on the values passed by c.
Except for refine which I agree should be set in SearchHandler, I would like order&sortType to be set in NewSearchForm. The reason is that I would like to reuse the SearchForm for things beside the Search pages (for example, in mod panel for listing pages).
Therefore, in order to not copy/paste multiple times the lines for "order" and "SortType", having them directly in NewSearchForm is better.